<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaddockWidgetBase</class>
 <widget class="QWidget" name="PaddockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="paddockListGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Paddocks</string>
      </property>
      <layout class="QVBoxLayout" name="paddocksVerticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PaddockLayerList" name="paddockList">
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="paddockFilterLabel">
           <property name="text">
            <string>Filter Paddocks by name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="paddockFilterLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="clearPaddockFilterButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Clear Paddock Filter</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../images/item-undo.png</normaloff>../../../images/item-undo.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="paddockLandSystemListGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Paddock Land Systems</string>
      </property>
      <layout class="QVBoxLayout" name="paddockLandSystemsVerticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PaddockLandSystemsLayerList" name="paddockLandSystemList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="spacerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaddockLayerList</class>
   <extends>QListView</extends>
   <header>mlapp/src/widgets/feature_list/paddock_layer_list</header>
  </customwidget>
  <customwidget>
   <class>PaddockLandSystemsLayerList</class>
   <extends>QListWidget</extends>
   <header>mlapp/src/widgets/feature_list/paddock_land_systems_layer_list</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
